16/danielmichaels/weather.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+import sys
+
+from api_secrets import WEATHER_TOKEN
+
+import requests
+import requests_cache
+
+from requests.exceptions import RequestException
+
+
+class Weather:
+    """A simple class that pulls in weather data from the openweathermap api and formats the response.
+
+    """
+
+    def __init__(self, city="london", country="gb"):
+        self.city = city
+        self.country = country
+        self.url = (
+            "http://api.openweathermap.org/data/2.5/weather?q={city},"
```

Comment:
f-string

Reply:
I've used f-strings below so unsure why I chose to adopt format string here.